common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +34,11 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            application_log.exception("UnicodeDecodeError Event-Data: %s", event)
```

Comment:
You probably need to use %r instead of %s.  Otherwise, I would have a concern that it wouldn't be able to write out the event data because of encoding issues.   But it would be good to confirm that in some python environment.